lib/vrl/compiler/src/value/convert.rs

```diff
@@ -230,6 +230,27 @@ impl Value {
             }),
         }
     }
+
+    /// Converts the Value into a byte representation regardless of it's original type.
+    /// Object and Array are currently not supported, although technically there's no reason why it
+    /// couldn't in future should the need arise.
```

Comment:
```suggestion
    /// Converts the Value into a byte representation regardless of its original type.
    /// Object and Array are currently not supported, although technically there's no reason why it
    /// couldn't in future should the need arise.
```

Reply:
I believe the apostrophe is correct. We are referring to the original type belonging to it. Thus it is a [possesive apostrophe](https://www.apostrophechecker.com/rules-for-possessive-apostrophes/), and should be included.

Reply:
Never thought about it that way, that would be definitely logical.

I think the possessive form with "it" is a special case, though:

> It’s is a contraction of “it is” or “it has.” Its is a possessive determiner we use to say that something belongs to or refers to something.

https://www.grammarly.com/blog/its-vs-its/

> It’s and its are among the most commonly confused words. They are pronounced the same, there’s a very small difference in how they’re written, _and it’s also easy to mistake the contraction in it’s for a possessive_.

Reply:
Well I never!